### ISSUE TITLE
sysctl: parameter value check for already defined params

### ISSUE THREAD

**Issue by hoall** [ansible/ansible-modules-core#4969]
##### ISSUE TYPE

- Feature Idea
##### COMPONENT NAME

sysctl
##### ANSIBLE VERSION

```

```
##### SUMMARY

Multiple roles/plays can modify the sysctl and set different values for the same parameter. My suggestion is to add a check functionality, that - if set to yes - checks if the new value is greater ( or less or equal) than the existing value. If so, it sets it, otherwise skip it.

The problem at the moment is that I can't check the value and set it in one task. It doesn't look clean at the moment and is just a workaround. example follows.

I know that this can't work for all parameter, because i.e. kernel.sem contains a string of parameters. But for the mayority it should be a benefit.
##### STEPS TO REPRODUCE

```
- name: check parameter fs.file-max
  shell: "sysctl -a | grep  fs.file-max | grep -o [0-9]*"
  args:
    executable: /bin/bash
  register: fs_file_max

- name: set parameter in sysctl fs.file-max
  sysctl:
    name="fs.file-max"
    value=6815744
    state=present
  when: fs_file_max.stdout <= 6815744

# Would be nice to have something like that:

- name: set parameter in sysctl fs.file-max
  sysctl:
    name="fs.file-max"
    value=6815744
    state=present
    check=yes
    check-params: ge
# ge - greater equal, le - less equal, e - equal
```

##### ACTUAL RESULTS

```

```

**Comment by ansibot** [comment]
@davixx ping, this issue is waiting for your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@davixx, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
This repository has been locked. All new issues and pull requests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pull request to the ansible/ansible repo.

**Comment by ansibot** [comment]
This issue was migrated to https://github.com/ansible/ansible/issues/29636
